packages/terra-progress-bar/docs/progress-bar-color.md

```diff
@@ -0,0 +1,41 @@
+## Progress Bar Colors
+
+By passing in a colorClass, authors can manually set the bar color of any `<ProgressBar />` component.
```

Comment:
Should we update this example to be locally scoped?

Reply:
I assumed not because I figured some uses of colorClass were going to involve reusing the  same classes between multiple progress bars. Though that might be because I've been staring at the example pages too much and they all have multiple bars on them.

Reply:
Since these are base css, we can't use the `:local` scope. I was concerned that the goal of these was to be as simple as possible for the examples, so turning them into `colors.modules.scss` might be more then necassary.